Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -2602,4 +2612,98 @@ class HTTPClientTests: XCTestCase {
 
         XCTAssertThrowsError(try future.wait())
     }
+
+    // MARK: - Tracing -
+
+    func testSemanticHTTPAttributesSet() throws {
+        let tracer = TestTracer()
+        InstrumentationSystem.bootstrap(tracer)
+
+        let localHTTPBin = HTTPBin(ssl: true)
+        let localClient = HTTPClient(eventLoopGroupProvider: .shared(self.clientGroup),
+                                     configuration: HTTPClient.Configuration(certificateVerification: .none))
+        defer {
+            XCTAssertNoThrow(try localClient.syncShutdown())
+            XCTAssertNoThrow(try localHTTPBin.shutdown())
+        }
+
+        let url = "https://localhost:\(localHTTPBin.port)/get"
+        let response = try localClient.get(url: url, context: testContext()).wait()
+        XCTAssertEqual(.ok, response.status)
+
+        print(tracer.recordedSpans.map(\.attributes))
+    }
+}
+
+private final class TestTracer: TracingInstrument {
+    private(set) var recordedSpans = [TestSpan]()
+
+    func startSpan(
+        named operationName: String,
+        context: BaggageContextCarrier,
+        ofKind kind: SpanKind,
+        at timestamp: Timestamp
+    ) -> Span {
+        let span = TestSpan(operationName: operationName,
+                            kind: kind,
+                            startTimestamp: timestamp,
+                            context: context.baggage)
+        recordedSpans.append(span)
+        return span
+    }
```

Comment:
the context should be updated, see https://github.com/slashmo/gsoc-swift-tracing/issues/126#issuecomment-674630622